public/app/routes/RoutesWrapper.tsx

```diff
@@ -13,6 +13,7 @@ import { AppChrome } from '../core/components/AppChrome/AppChrome';
 import { AppNotificationList } from '../core/components/AppNotifications/AppNotificationList';
 import { ModalsContextProvider } from '../core/context/ModalsContextProvider';
 import { useSidecar } from '../core/context/SidecarContext';
+import { QueriesDrawerContextProvider } from '../features/explore/QueriesDrawer/QueriesDrawerContext';
```

Comment:
`QueriesDrawerContextProvider` is Explore specific but `RoutesWrapper` is more generic. Do we need to move the Context here? 🤔

Reply:
it needs to be higher than `AppChrome` so that the button can access the context when present in `AppChrome`. 

i'm not overly worried about this tbh, `QueriesDrawerContextProvider` is Explore-specific _right now_, but there are plans to support query library in other places (e.g. dashboard panels). once we get to that point, it makes sense for it to be a global context provider anyway 😄